app/controllers/admin/navigation_links_controller.rb

```diff
@@ -0,0 +1,49 @@
+module Admin
+  class NavigationLinksController < Admin::ApplicationController
+    ALLOWED_PARAMS = %i[
+      name url icon requires_auth
+    ].freeze
+    layout "admin"
+
+    def index
+      @navigation_links = NavigationLink.all.order(:name)
```

Comment:
FYI `.all` is redundant in this case:

```ruby
[5] pry(main)> NavigationLink.all.order(:name).to_sql
=> "SELECT \"navigation_links\".* FROM \"navigation_links\" ORDER BY \"navigation_links\".\"name\" ASC"
[6] pry(main)> NavigationLink.order(:name).to_sql
=> "SELECT \"navigation_links\".* FROM \"navigation_links\" ORDER BY \"navigation_links\".\"name\" ASC"
```

Reply:
Urgh I've done this before and you corrected it, and I did it again 🤦🏽‍♀️ ✅